packages/components/tests/dummy/app/templates/components/alert.hbs

```diff
@@ -220,11 +220,11 @@
       <p>It is a very simple component that yields its content.</p>
       <p><em>Notice: generic the content will appear at the bottom, after title, description and actions, and the
           developer will need to take care of spacing, layout and styling of the custom content in this case.</em></p>
-      <p>🚨
-        <em><strong>Important</strong>: this method should be used only in special cases and as an escape hatch. If you
-          find yourself in need to use it, we suggest to speak with the design system team to check that the solution is
-          conformant and satifies the accessibility criteria.
-        </em></p>
+      <p>
```

Comment:
I see others have a class `dummy-paragraph` applied to them, does this one need it too?

```suggestion
      <p class="dummy-paragraph">
```

(probably irrelevant for your parser/converter, but just in case)

Reply:
yes, there are a few instances missing the `dummy-paragraph` class, will add them as it's rather easy, but won't have any impact on the new website indeed